ui/src/components/CountrySelector.js

```diff
@@ -91,8 +91,11 @@ export class CountrySelector extends LitElement {
      * Emits an event that causes the country selector to close and the appropriate
      * Terms of Service to appear. 
      */
-    handleOpenTerms() {
-        const event = new CustomEvent('open-tos', {});
+    handleCountrySelected() {
+        const event = new CustomEvent('open-tos', {
```

Comment:
nit change name of event to `country-selected` and include country in the custom event detail. So then you don't need a getter for country either. It comes with the event.

Reply:
adjusted